src/Config/ObjectModel/CliBool.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.DataApiBuilder.Config.ObjectModel
+{
+    public enum CliBool
+    {
+        // The enum value None is required to determine whether a value was provided for a CLI option. In case no value is provided for the option,
```

Comment:
why we need `None`, if the user doesn't want to provide a value, they can just not use the option.

Reply:
This is because of opposite nature of `rest.disabled` and `rest.enabled` options. 
Assume a dab init command where `rest.disabled` is included in the init command and `rest.enabled` is omitted.
Since `rest.disabled` is included, it means we want to disable the REST endpoint. 
Now for `rest.enabled` we have an enum. And when we do not provide a value for an enum, it assumes the default value of first enum value. If don't include `None`, it would assume the value of True/False depending on whichever is written first in the enum declaration. This is not what we want. We want to explicitly know that the `rest.enabled` option was not included in the init command and no consideration should be given to it in the case when it is coupled with `rest.disabled` flag. Exclusion of `rest.enabled` coupled with inclusion of `rest.disabled` means the user is not bothered about the behavior of `rest.enabled` and he would be assuming that the rest endpoint will be disabled.